Src/FluentAssertions/CallerIdentification/ShouldCallParsingStrategy.cs

```diff
@@ -4,7 +4,7 @@ namespace FluentAssertions.CallerIdentification
 {
     internal class ShouldCallParsingStrategy : IParsingStrategy
     {
-        private const string ShouldCall = ".Should()";
+        private const string ShouldCall = ".Should(";
```

Comment:
Is there any disadvantage to using `.Should(` over `.Should()`?
I imagine `Should` is only called with an argument in our tests when we fake timing.

Reply:
It _think_ it seems reasonable  this way it also still works if somebody were to overload `.Should` via an extension method (not sure why they would though).